src/components/header/header.scss

```diff
@@ -1,62 +0,0 @@
-
```

Comment:
🙌 Desktop is looking good in Chrome. Will just later need small adjustments for other browsers like Safari (for example, the shadows are not adjusting the gradient).

Read up on Mobile-first development and media queries. Once we've implemented it, I think we'll be ok to put this up online. Basically, we want to develop for the smallest smartphone screens first and then grow from there, applying media queries for bigger and bigger screens on top of the smaller screen sizes. Use the browser dev-tools to preview what it'll look like on the different screen sizes.
<img width="1103" alt="Screen Shot 2020-05-19 at 9 51 39 AM" src="https://user-images.githubusercontent.com/14047327/82355640-573e7f80-99b7-11ea-99ae-aadaa7b89afc.png">


Check out this line. Previous to this, the queries above it will support small screens. This is a breakpoint where we start to support a screen that's a bit bigger. https://github.com/nourimeals/website/blob/4e8def8ea3f7422b4a4d4e883f6755e02c2f5c96/src/components/home/home.scss#L51